### PR TITLE
Feature/user review

### PR DESCRIPTION
**Pull request by AlexeyTrekin**
- add user reviews: accept/reject with text message and (optional) vector layer
- processings table columns can be hidden by user (see settings tab)
- processings in which reviews soon expire, are hightlighted

### CODE REVIEW

File: mapflow/config.py [hunk 1]
```diff
@@ -16,9 +16,30 @@ class Config:
...
+                                'percentCompleted': self.tr("Progress %"),
+                                'aoiArea': self.tr("Area, sq.km"),
+                                'cost': self.tr("Cost"),
+                                'created': self.tr("Created"),
+                                'reviewUntil': self.tr("review until"),
+                                'id': self.tr("ID")}
+    """
+    PROCESSING_TABLE_ID_COLUMN_INDEX = PROCESSING_TABLE_COLUMNS.index('id')
+    PROCESSING_TABLE_SORT_COLUMN_INDEX = PROCESSING_TABLE_COLUMNS.index('created')
+    DEFAULT_HIDDEN_COLUMNS = (PROCESSING_TABLE_COLUMNS.index(item) for item in ('id', 'reviewUntil'))
```
Comment (Godnik) [inline mapflow/config.py:42]: 'cost' field is also hidden by default?

File: mapflow/dialogs/static/ui/main_dialog.ui [hunk 2]
```diff
@@ -1261,6 +1292,107 @@
              </property>
             </spacer>
            </item>
+           <item>
```
Comment (Godnik) [inline mapflow/dialogs/static/ui/main_dialog.ui:1295]: Vertical widget is better

File: mapflow/dialogs/static/ui/review_dialog.ui
Comment (Godnik) [inline mapflow/dialogs/static/ui/review_dialog.ui:1]: The appearance of the dialogue should be connected to the review option (config?), not added by default

File: mapflow/config.py
Comment (Godnik) [inline mapflow/config.py:1]: We need some setting as `feedback_form_mode = review #rating`
The default mode should be `rating` as `review` is available for commercial users by activation

File: mapflow/config.py (hunk 1 above)
Comment (AlexeyTrekin) [reply mapflow/config.py:42]: Yes, looks reasonable, will add.

File: mapflow/dialogs/static/ui/main_dialog.ui (hunk 2 above)
Comment (AlexeyTrekin) [reply mapflow/dialogs/static/ui/main_dialog.ui]: Merged

File: mapflow/config.py
Comment (AlexeyTrekin) [reply mapflow/config.py:1]: It is here https://github.com/Geoalert/mapflow-qgis/blob/c94e7e1db050a427c692cc63138cb99c610bbcb2/mapflow/mapflow.py#L75**](https://github.com/Geoalert/mapflow-qgis/blob/c94e7e1db050a427c692cc63138cb99c610bbcb2/mapflow/mapflow.py#L1690

We setup it on first request to /user/status API, not from local settings, as it depends not on the client, but on the server-side

File: mapflow/dialogs/static/ui/review_dialog.ui
Comment (AlexeyTrekin) [reply mapflow/dialogs/static/ui/review_dialog.ui:1]: It is connected to the user account settings on server 
https://github.com/Geoalert/mapflow-qgis/blob/c94e7e1db050a427c692cc63138cb99c610bbcb2/mapflow/mapflow.py#L1690

**Pull request merged**